### ISSUE TITLE
RC position sanity incorrect for LON

### ISSUE THREAD

**Issue by marcus-aa** [mutability/mlat-client#34]
https://github.com/mutability/mlat-client/commit/55cf9809e8b7f9fa2330ae84b89e66b2be20e29a

row 337, isn't the sanity check wrong? Excludes all longitudes except 180 exactly?

if lat >= -90 and lat <= 90 and lon >= -180 and lon <= -180:

should be 

if lat >= -90 and lat <= 90 and lon >= -180 and lon <= 180:

**Comment by mutability** [comment]
Yeah, looks wrong. piaware doesn't actually rely on this path which is
possibly why I never noticed ..
